apps/authentication/mfa/face.py

```diff
@@ -32,7 +32,9 @@ def is_active(self):
 
     @staticmethod
     def global_enabled():
-        return settings.XPACK_LICENSE_IS_VALID and settings.FACE_RECOGNITION_ENABLED
+        return settings.XPACK_LICENSE_IS_VALID \
+            and settings.XPACK_LICENSE_EDITION == 'ultimate' \
+            and settings.FACE_RECOGNITION_ENABLED
 
     def get_enable_url(self) -> str:
         return reverse('authentication:user-face-enable')
```

Comment:
To ensure proper formatting of the output and enhance readability, it's recommended to maintain consistent indentation levels when writing comments and documentation strings. Here's how you might reformat the code:

```python
def is_active(self):

     # Check XPACK license validity and facial recognition setting

@staticmethod
def global_enabled():
    return (
        settings.XPACK_LICENSE_IS_VALID
        and settings.FACE_RECOGNITION_ENABLED
    )

def get_enable_url() -> str:
    
    url = reverse('authentication:user-face-enable')

class UserFaceEnableView(View):
    ...
```
Changes made include:
- Ensuring consistent spacing around operators (e.g., `settings.` instead of just `,`)
- Adding space before opening parenthesis in conditional statement (`and`, not `or`)

Optimization suggestions could be improving method names if they seem overly verbose, but the context doesn't provide information about which methods should be optimized.

For instance, consider replacing multiple checks in `global_enabled()` with an expression that allows for short-circuit evaluation, such as `lambda: ...`.